packages/block-library/src/embed/embed-preview.native.js

```diff
@@ -91,13 +101,17 @@ const EmbedPreview = ( {
 						}
 					} }
 				>
-					<View pointerEvents="box-only">
+					<View
+						pointerEvents="box-only"
+						style={ [ wrapperStyle, wrapperAlignStyle ] }
+					>
 						<SandBox
 							html={ html }
 							title={ iframeTitle }
 							type={ sandboxClassnames }
 							providerUrl={ providerUrl }
 							url={ url }
+							containerStyle={ sandboxAlignStyle }
```

Comment:
This new prop allows us to control the styles of the WebView component, for this case, we're passing the styles required to align the content.